modules/bmo-about-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,71 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: REFERENCE
+[id="about-the-hostfirmwaresettings-resource_{context}"]
+= About the HostFirmwareSettings resource
+
+You can use the `HostFirmwareSettings` resource to retrieve and manage the BIOS settings for a host. When a host moves to the `Available` state, Ironic reads the host's BIOS settings and creates the `HostFirmwareSettings` resource. The resource contains the complete BIOS configuration returned from the baseboard management controller (BMC). Whereas, the `firmware` field in the `BareMetalHost` resource returns three vendor-independent fields, the `HostFirmwareSettings` resource typically comprises many BIOS settings of vendor-specific fields per host.
+
+The `HostFirmwareSettings` resource contains two sections:
+
+. The host firmware specification.
+. The host firmware's current status.
+
+[discrete]
+== The `HostFirmwareSettings` specification
+
+The `spec` section of the `HostFirmwareSettings` resource defines the desired state of the host's BIOS. Ironic uses the settings in the `spec` section to update the baseboard management controller (BMC) when the host is in the `Preparing` state. The `spec` section uses the `FirmwareSchema` resource schema to ensure that you do not send invalid name/value pairs to hosts. See "About the FirmwareSchema resource" for additional details.
+
```

Comment:
I would add here that by default the section is empty.
Also May be it'd be good to mention that also Integer parameters are listed in the status section as strings, i.e. "1", in spec section they should be set as integers, without quotes.
@bfournie WDYT?